adapters/verlag-adapter/src/main/java/de/adorsys/xs2a/adapter/service/impl/VerlagAccountInformationService.java

```diff
@@ -79,32 +77,4 @@ private RequestHeaders modifyAcceptHeader(RequestHeaders requestHeaders, String
         headersMap.put(RequestHeaders.ACCEPT, acceptHeader);
         return RequestHeaders.fromMap(headersMap);
     }
-
-    @Override
-    protected Map<String, String> populatePostHeaders(Map<String, String> headers) {
-        addApiKey(headers);
-        return headers;
-    }
```

Comment:
The same as with `date` header: I'd better go with the old approach of duplicates: it is more readable and maintainable.

Reply:
Reverted.